components/src/core/mobile-stepper/mobile-stepper.test.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { MobileStepper } from '.';
+import TestRenderer, { ReactTestInstance } from 'react-test-renderer';
+import { ProgressBar } from 'react-native-paper';
+import { View } from 'react-native';
+
+describe('MobileStepper', () => {
+    it('should render the correct number of steps', () => {
```

Comment:
May want a few edge case tests here...what happens if they set steps={-5} or steps={0} or activeStep={null}

Reply:
are we able to set steps based on the active steps with the knobs? If steps are set to 4, active to be the same...
![image](https://user-images.githubusercontent.com/37150565/104458768-ca65b900-5579-11eb-8614-9e6761865fb1.png)

Reply:
We can bound it in the component so that active step is never higher than the number of steps. Dynamically changing the knobs in the storybook is more challenging though...Tom tried it and ran into some issues getting it to work, so we backed down on that for now.